### DISCUSSION TITLE
Do I have to give you all the information about my family?

### DISCUSSION THREAD

**Discussion by Prixii** [community/community#50989] [GitHub Education]
You are significantly more likely to be verified if you have completed your User Profile with your full name as it appears in your academic affiliation document plus a short bio.
I've been submitted 21 requests and was rejected 21 times!
DISAPPOINT

**Comment by ideviceuser** [comment]
feel you bro. 16 request all declined within minutes. Tried school ID, scheduled class table and monthly bill.
I'm only getting generic errors and no real explanation what I need to do.

**Comment by samus-aran** [comment]
Hi @Prixii 

Often this common pre-processing warning is displayed because the application detects differences between your profile information and the evidence you’ve supplied. We do request that students can demonstrate that their GitHub account corresponds with the evidence given. The verification process looks at your GitHub username, email, and name (as well as other points) to cross-reference the evidence submitted, so be sure that they match to ensure a smoother process.

We would also suggest that when you update your profile information that you log out and log back into GitHub before re-applying.
